src/test/java/kitchenpos/application/OrderTableServiceTest.java

```diff
@@ -0,0 +1,229 @@
+package kitchenpos.application;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import java.util.UUID;
+import kitchenpos.domain.OrderRepository;
+import kitchenpos.domain.OrderStatus;
+import kitchenpos.domain.OrderTable;
+import kitchenpos.domain.OrderTableRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class OrderTableServiceTest {
+
+    @Mock
+    private OrderTableRepository orderTableRepository;
+    @Mock
+    private OrderRepository orderRepository;
+
+    private OrderTableService orderTableService;
+
+    @BeforeEach
+    void setUp() {
+        orderTableService = new OrderTableService(orderTableRepository, orderRepository);
+    }
+
+    @DisplayName("사용자는 `주문 테이블`을 생성할 수 있다.")
+    @Test
+    void create() {
+        // given
+        final OrderTable orderTable = new OrderTable();
+        orderTable.setName("테이블");
+
+        when(orderTableRepository.save(any())).then(invocation -> {
+            OrderTable savedOrderTable = invocation.getArgument(0);
+            savedOrderTable.setId(UUID.randomUUID());
+            return savedOrderTable;
+        });
+
+        // when
+        OrderTable createdOrderTable = orderTableService.create(orderTable);
+
+        // then
+        assertAll(
+            () -> assertNotNull(createdOrderTable),
+            () -> assertNotNull(createdOrderTable.getId()),
+            () -> assertEquals(orderTable.getName(), createdOrderTable.getName()),
+            () -> assertEquals(0, createdOrderTable.getNumberOfGuests()),
+            () -> assertFalse(createdOrderTable.isOccupied())
+        );
+    }
+
+    @DisplayName("`주문 테이블`을 생성할 때 이름이 없으면 예외가 발생한다.")
+    @Test
+    void createWithEmptyName() {
+        // given
+        final OrderTable orderTable = new OrderTable();
+        orderTable.setName("");
+
+        // when & then
+        assertThrows(IllegalArgumentException.class, () -> orderTableService.create(orderTable));
```

Comment:
assertThrows도 나쁘지 않지만 저는 개인적으로 assertJ의 `assertThatThrownBy`가 조금 더 직관적이여서 이 검증문을 조금 더 선호하는 편이에요. 🙂

```suggestion
        assertThatThrownBy(() -> orderTableService.create(orderTable))
            .isExactlyInstanceOf(IllegalArgumentException.class);
```

Reply:
OrderTableService test 쪽만 assertThatThrownBy() 로 수정해보았는데 가독성면에서 더 편한것 같습니다! 다음 테스트코드에서는 assertThatThrownBy() 지향해보겠습니다 : )